docs/rules/no-unused-vars.md

```diff
@@ -182,6 +182,63 @@ function foo(x, _y) {
 foo();
 ```
 
+### exceptions
+
+The `exceptions` option is used for `catch` block arguments validation.
+
+It has two settings:
+
+* `none` - do not check arguments. This is the default setting.
+* `all` - all named arguments must be used.
+
+#### exceptions: none
+
+Not specifying this rule is equivalent of assigning it to `none`.
+
+Examples of **correct** code for the `{ "exceptions": "none" }` option:
+
+```js
+/*eslint no-unused-vars: [2, { "exceptions": "none" }]*/
+
+try {
+    //...
+} catch( err ){
+    console.error("exception");
```

Comment:
@platinumazure Just to clarify, did you mean `catch (){` or `catch() {`? I'm assuming you meant the latter, but could see the suggestion being taken either way  :)